bundles/org.openhab.binding.bluetooth/src/main/java/org/openhab/binding/bluetooth/ConnectedBluetoothHandler.java

```diff
@@ -192,7 +193,7 @@ public void onCharacteristicReadComplete(BluetoothCharacteristic characteristic,
                 updateBatteryLevel(characteristic);
             } else {
                 logger.debug("Characteristic {} from {} has been read - value {}", characteristic.getUuid(), address,
-                        characteristic.getValue());
+                        HexUtils.bytesToHex(characteristic.getByteValue()));
```

Comment:
These calls are quite expensive, I don't know how often this is called but we might want to wrap it in a check to `logger.isDebugEnabled`

Reply:
done